src/util/helpers.tsx

```diff
@@ -298,3 +299,15 @@ export const getFileExtension = (filename: string): string => {
 
   return `.${filename.split(".").pop()}` || "";
 };
+
+export const getDevices = (
```

Comment:
The name is misleading. Maybe return all devices here, and move the filter to the use case?

So the funciton body here becomes
```
  return Object.entries(
    instance.expanded_devices ?? {},
  );
```